packages/service-library/tests/rabbitmq/test_rabbitmq.py

```diff
@@ -435,3 +435,20 @@ async def test_rabbit_not_using_the_same_exchange_type_raises(
     # now do a second subscribtion wiht topics, will create a TOPICS exchange
     with pytest.raises(aio_pika.exceptions.ChannelPreconditionFailed):
         await client.subscribe(exchange_name, mocked_message_parser, topics=[])
+
+
+@pytest.mark.no_cleanup_check_rabbitmq_server_has_no_errors()
```

Comment:
that is an interesting an fun decorator... why is that?

Reply:
Yes, we can check together